src/screens/Profile/index.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { SafeAreaView } from 'react-native';
+
+import Header from './components/Header';
+import SwitchSelector from './components/SwitchSelector';
+import { Post } from '../../components/Post/Post';
+
+import { Container, Name, Biography } from './styles';
+
+const Profile: React.FC = () => {
+  return (
+    <Container>
+      <SafeAreaView style={{ flex: 1 }}>
+        <Header
+          background_photo="https://images.pexels.com/photos/1005417/pexels-photo-1005417.jpeg?auto=compress&cs=tinysrgb&h=750&w=1260"
+          profile_image="https://images.pexels.com/photos/2666467/pexels-photo-2666467.jpeg?auto=compress&cs=tinysrgb&h=750&w=1260"
+        />
+
+        <Name>Arwen Undómiel</Name>
+        <Biography>
```

Comment:
do a check to see if anything was passed in the biography property, and render only if they pass something, because the bio is something that can be null